suite-common/trading/src/selectors/tradingSelectors.ts

```diff
@@ -111,12 +148,23 @@ export const selectTradingBuyProviders = createMemoizedSelector(
     buyInfo => buyInfo?.providerInfos,
 );
 
+export const selectTradingExchangeProviders = createMemoizedSelector(
```

Comment:
this selector is not used except in tests, shouldn't we replace it just like `selectTradingBuyProviders`?